app/models/notification.rb

```diff
@@ -0,0 +1,35 @@
+# == Schema Information
+#
+# Table name: notifications
+#
+#  id                   :bigint           not null, primary key
+#  notification_type    :integer          not null
+#  primary_actor_type   :string           not null
+#  read_at              :datetime
+#  secondary_actor_type :string
+#  created_at           :datetime         not null
+#  updated_at           :datetime         not null
+#  account_id           :bigint           not null
+#  primary_actor_id     :bigint           not null
+#  secondary_actor_id   :bigint
+#  user_id              :bigint           not null
+#
+# Indexes
+#
+#  index_notifications_on_account_id               (account_id)
+#  index_notifications_on_user_id                  (user_id)
+#  uniq_primary_actor_per_account_notifications    (primary_actor_type,primary_actor_id)
+#  uniq_secondary_actor_per_account_notifications  (secondary_actor_type,secondary_actor_id)
+#
+
+class Notification < ApplicationRecord
```

Comment:
Should we make it push notification as we already have an Email Notification

Reply:
The other table is Notification Settings where we save the settings for Email, we will add an additional attribute to settings table to store the push notification settings. 

we will create an entry in notification table for each of the notifiable activities.